Versioning.md

```diff
@@ -65,6 +65,12 @@ Refer to the [ABI Policy](./docs/abi-policy.md) for more details. To summarise:
   allowed to break existing stable interfaces. Feature flags will be removed
   once we have a stable implementation for the signal.
 
+* As an exception, small experimental features in otherwise stable signals/components
+  mayn't necessarily be released under feature flag. These would be flagged as experimental
+  by adding a `NOTE` in it's header file - either at the beginning of file, or as the comment for
```

Comment:
non-blocking comment -

Would it make sense to call the header file `experimental_*` to further clarify?

I think you could do this and still provide backwards compatibility later by forward-chain including the new non-experimental folder when it stabilizes.   Just a thought.

Reply:
Yes this does makes sense. Will incorporate this.

Reply:
This is done now.